Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Builder
+FROM golang:1.14.9 as builder
```

Comment:
Why not 1.15?

Reply:
This was a guess based on the current go.mod file

https://github.com/filecoin-project/sentinel-visor/blob/47b18855ece919e6c8d337b2a7483764f928090c/go.mod#L3

Reply:
go.mod just defines the minimum version, so it's ok to use a later one